Winleafs.Wpf/Views/App.xaml

```diff
@@ -13,6 +13,8 @@
                 <ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Defaults.xaml" />
                 <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Primary/MaterialDesignColor.DeepPurple.xaml" />
                 <ResourceDictionary Source="pack://application:,,,/MaterialDesignColors;component/Themes/Recommended/Accent/MaterialDesignColor.Lime.xaml" />
+                <!--Toastr notifications-->
+                <ResourceDictionary Source="pack://application:,,,/ToastNotifications.Messages;component/Themes/Default.xaml" />
```

Comment:
You should also add this to DesignTimeResources.xaml

Reply:
Doesn't show up at design time so it's not needed there, these can only be triggered with code.

Reply:
True, but I'd rather include it and keep the files identical, for completeness